GameWordPicker.java

```diff
@@ -7,7 +7,9 @@
 import java.util.Random;
 import java.util.Scanner;
 
-public class GameWordPicker extends GameScene {
+public class GameWordPicker {
```

Comment:
Changed GameWordPicker to independent class since there will be a conflict if it derives GameScene.